src/tir/ir/sparse.cc

```diff
@@ -43,18 +43,20 @@ TVM_REGISTER_GLOBAL("tir.sparse.GetAxisIndexType").set_body_typed([](Axis axis)
 });
 
 // DenseFixedAxis
-DenseFixedAxis::DenseFixedAxis(String name, PrimExpr length) {
+DenseFixedAxis::DenseFixedAxis(String name, PrimExpr length, Optional<SparseAxis> from_sparse) {
```

Comment:
Why do we need to record where does come from?

Reply:
Explained here https://github.com/yzh119/tvm/pull/14#discussion_r745703781 :eyes: